nextcord/components.py

```diff
@@ -77,6 +77,10 @@ def __repr__(self) -> str:
     def _raw_construct(cls, **kwargs) -> Self:
         self = cls.__new__(cls)
         for slot in get_slots(cls):
+            if slot == "type" and getattr(self, slot, None) is not None:
+                # nextcord/issues/1103
+                continue
```

Comment:
Wasn't this fixed by #1104?

Reply:
Possibly? I haven't been keeping track on more recent PRs. Please let me know what I should do with this code, provided changes are in order :)

Reply:
All that PR did when it was merged was remove `"type"` from the `__slots__` of the `Component` class. As long as your fork is up to date with the upstream and you remove this code I commented on, all should be good.